qemu/tests/cfg/timerdevice.cfg

```diff
@@ -64,6 +64,7 @@
                     variants:
                         - without_host_load:
                         - with_host_load:
+                            login_timeout = 1200
```

Comment:
1200s is a quite long time. 
How did you get it?

Reply:
Thanks for review, I changed another RHEL.7.5 guest, this time can be smaller. But for windows guest, it should be much longer than linux guest, even though the `timerdevice_boot.py` didn't implement windows part now.